app/views/appointments/calendar.html.erb

```diff
@@ -2,28 +2,35 @@
   <div class="container mt-5 text-center">
     <h1>Agenda</h1>
     <%= link_to "Nueva cita", new_appointment_path, class:"btn btn-outline-secondary btn-sm" %>
-    <%= week_calendar(events: @appointments) do |date, appointments| %>
-      <%= date.day %>
 
-      <% appointments.each do |appointment| %>
-        <div class="event">
-          <div class="card-header">
-            <%= link_to appointment.title, appointment_path(appointment) %>
-          </div>
-          <div class="card-body">
-            <p>
-              <%=appointment.description%>
-            </p>
-          </div>
-          <div class="cars-footer">
-            <p class="card-text">
-              <%= appointment.start_time.strftime("%I:%M %p") if appointment.start_time %>
-              <%= appointment.end_time.strftime("- %I:%M %p") if appointment.end_time %>
-            </p>
-          </div>
+    <%= week_calendar(events: @appointments) do |date, appointments| %>
```

Comment:
Is necessary to use "=" here? I think this look like a cicle not a element to print.